repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json

```diff
@@ -1,22 +1,22 @@
 {
-  "name": "ReposeFakeKeystonev2",
-  "version": "0.1.0",
-  "description": "Mock Keystone Service",
-  "readme": "README.md",
```

Comment:
I'm going to answer all of these at once.

I just copypasta'd the package.json file.

I'll fix it up.